doc/userguide/install.rst

```diff
@@ -16,6 +16,9 @@ Source
 
 Installing from the source distribution files gives the most control over the Suricata installation.
 
+The Suricata source tarballs should be verified before building the source, see
+:doc:`source-verification`
```

Comment:
Can we use common naming here?  "source distribution" or "source tarball"?

Reply:
Ah I hadn't paid attention to the previous lines, will update to match in the next PR. Any preference on wording?

Reply:
Not a big fan of "tarball" in official documentation.  I prefer "archive" or "distribution archive", etc.

Reply:
Alright will get that updated/reworked, thanks!

Reply:
nit: missing the period.